### DISCUSSION TITLE
Neustart aus Webui nicht mehr möglich

### DISCUSSION THREAD

**Discussion by ms-ch** [OpenCCU/OpenCCU#1318] [🇩🇪 Fragen&Antworten]
Seit letztem Update muss RaspberryMatic via ssh neu gestartet werden. Neustart via Button im Webui aktualisiert zwar die Webseite nach der Bestätigung zum Neustart. Jedoch ein Neustart findet nie statt.

**Comment by jens-maus** [comment]
Das kann ich selbst leider nicht bestätigen. Bitte mehr Infos zur verwendeten Hardwareplatform usw. Und wenn möglich mal via HDMI Monitor beobachten was bei einem Neustart via WebUI auf der Console so passiert.

**Comment by ms-ch** [comment]
Guten Tag Miteinander

Wie beschrieben, rebootet RaspberryMatic, resp. der Raspberry Pi  nicht, wenn via Webui ausgelöst. Auch wenn dieser Schritt mehrmals und in unterschiedlichen Browsern durchgeführt wird, rebootet der Raspberry nicht. Nur via ssh. Dieses Phänomen ist erst seit der aktuellsten Version der Fall, war vorher nie so aufgetreten und wurde bereits nach dem Softwareupdate festgestellt. Aber erst jetzt kommuniziert.

Anbei die Logs (messages.zip) der Console, seit Neustart via ssh von gestern Abend bis und mit Neustart von heute via Webui um 14.41 Uhr. 
In der Console war via Bildschirm aus meiner Sich nichts besonderes erkennbar, einzig es findet kein reboot statt.

Zusätzlich die Datei (Messages-boot.log) nach einem erfolgreichen reboot via SSH.

Gerät:	Raspberry Pi 4 Model B Rev 1.4, 8 GB
		RPI-RF-MOD, Geräte laut Geräte.pdf

***@***.***:~# uname -a
Linux raspberrymatic.domain-swiss.ch 5.10.17 #1 SMP PREEMPT Tue May 25 10:38:35 UTC 2021 aarch64 GNU/Linux

Falls noch weitere Angaben benötigt werden, bitte ungeniert nachfragen.

**Reply by jens-maus** [reply]
Tut mir leid, aber wie gesagt kann ich dieses verhalten nicht reproduzieren. Hier lässt sich ein RaspberryPi4 unter RaspberryMatic problemlos via WebUI rebooten. Es müsste hier also erst ein entsprechendes Debugging bzw. Analysen stattfinden um herauszubekommen ob das wirklich ein generelles Problem ist oder (wie vermutet) nur ein lokales Problem bei dieser Installation, denn die WebUI macht auch nichts anderes als ein /sbin/reboot aufzurufen wie man das auch via SSH tut. Bitte daher einmal in die Browser-Console schauen ob ein Fehler zu finden ist wenn man versucht via Webbrowser den Neustart auszulösen.
